README.md

```diff
@@ -2,15 +2,16 @@
 
 [![Slack](https://img.shields.io/badge/slack-gophers-green.svg?style=flat)](https://gophers.slack.com/messages/vscode/) [![Build Status](https://travis-ci.org/Microsoft/vscode-go.svg?branch=master)](https://travis-ci.org/Microsoft/vscode-go)
 
+> We're moving! Read more [here](#we-are-moving---) and subscribe to [microsoft/vscode-go#3247](https://github.com/microsoft/vscode-go/issues/3247) for updates.
```

Comment:
I am generally wary of links with link text that do not give an indication of what the link is for. Screenreader users have the feature to browse through the anchor links on a page and such link texts doesnt tell them if it is relevant to them or not

```suggestion
> Read the [We are moving 🏡 🚚 🏡](#we-are-moving---) section to learn about the new home for this extension and subscribe to [Microsoft/vscode-go#3247](https://github.com/microsoft/vscode-go/issues/3247) for updates.
```